runtime/j9jitbuilder/module.xml

```diff
@@ -24,7 +24,7 @@
 
 <module>
 	<artifact type="shared" name="j9jitbuilder">
-		<include-if condition="spec.flags.J9VM_OPT_PANAMA"/>
+		<include-if condition="spec.flags.opt_jitserver"/>
```

Comment:
I think I want to take back my earlier suggestion. This module is for the jit 'builder', which is independent of the jit 'server', so I now think this line should just be removed. If always including it is problematic, we can work out what the correct inclusion condition should be.

@mstoodle This module does not appear to be included in cmake builds: should it be?

Reply:
Let me know what should I do. If to remove the block or the file.

Reply:
Apparently we can't just remove the condition; it doesn't compile as is on Linux x64:
```
JavaMethodCallbackGen.cpp:25:10: fatal error: Jit.hpp: No such file or directory
 #include "Jit.hpp"
          ^~~~~~~~~
```
The only `Jit.hpp` I can find is `omr/fvtest/compilertest/Jit.hpp` which suggests this is (old?) test material. We'll need to await an answer from @mstoodle.

Reply:
I believe @tajila created this directory a couple of years back to experiment with the use of OMR's JitBuilder library in OpenJ9. Since we have not completed the integration work required to compile the JitBuilder library into OpenJ9 (see https://github.com/eclipse-openj9/openj9/issues/5836), I don't think this should be included.

@tajila do you concur?

Reply:
Yes, this does not need to be included.

Reply:
Thanks, @mstoodle and @tajila. @ohbus Please proceed by removing the folder `runtime/j9jitbuilder` and its contents.

Reply:
Thank you so much, everyone!